app/controllers/ping.js

```diff
@@ -0,0 +1,9 @@
+const PingController = function(app, pkg){
```

Comment:
@rodrigopavezi Also wondering if this should be in a `routes` folder instead of `controllers`? I could visualize having a `FileController` that has functions for `download`, `handleError`, and `getFileName` as in my other PR, but I'm not sure that the route handler would go inside there as well. Thoughts?